api/valor_api/backend/metrics/detection.py

```diff
@@ -98,37 +103,56 @@ def _compute_curves(
         for confidence_threshold in [x / 100 for x in range(5, 100, 5)]:
             if grouper_id not in sorted_ranked_pairs:
                 # happens when there are no ground truths for a label
-                tp = 0
+                tp = []
                 fn = (
```

Comment:
Shouldn't this be a false-positive if there is no groundtruth for the label?

Reply:
Is grouper_id indicative of groundtruths or predictions?

Reply:
`sorted_ranked_pairs` contains ground truth + prediction matches. if there's no grouper_id in that object, then there weren't any qualifying ground truth + prediction matches for that grouper label (and hence no true positives)

Reply:
I agree that comment is confusing, I'll remove it